README.md

```diff
@@ -49,6 +49,34 @@ npx hardhat deploy  --network localhost
 
 Hardhat re-uses old deployments, to force re-deploy add the `--reset` flag above
 
+## ZKLLVM output check
+
+Place folder with ZKLLVM output to `contracts/zkllvm` directory.
+
+Deploy contracts
+```
+npx hardhat deploy
+```
+
+Verify all folders from `contracts/zkllvm` director
+```
+npx hardhat verify
+```
+
+Verify one folder from `contracts/zkllvm` directory
+```
+npx hardhat verify-one folder-name
+```
+
+ZKLLVM output folder format
```

Comment:
Let's put this into the beginning of the subsection. Because other commands don't make sense if you don't know what is ZKLLVM output folder.